src/runtime-rs/crates/hypervisor/src/device/tap.rs

```diff
@@ -0,0 +1,448 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Duplicated line? Btw, do we need to append a copyright of Kata Containers? /cc @Apokleos

Reply:
Yes, agree. FYI @lifupan